biosimulations/apps/dispatch-service/src/app/app.controller.ts

```diff
@@ -130,15 +140,40 @@ export class AppController {
                     });
                   });
                 });
+              })
+              .on('error', (err) => {
+                console.log('Error occured in file writing', err);
               });
           }
         }
-      })
-
-      
+      });
     }
   }
 
+  @MessagePattern('dispatch_log')
```

Comment:
When is the "dispatch_log" message sent? Should this not be listening for the normal dispatch message?